qiskit/circuit/library/n_local/qaoa_ansatz.py

```diff
@@ -38,7 +34,7 @@ def __init__(
         reps: int = 1,
         initial_state: Optional[QuantumCircuit] = None,
         mixer_operator=None,
-        name: str = "qaoa",
+        name: str = "QAOA",
```

Comment:
Is this a potentially breaking change for people if they're matching on the default name?